src/tasks/match_modalities/metrics/knn_auc/config.vsh.yaml

```
@@ -3,12 +3,12 @@ functionality:
   name: "knn_auc"
   info:
     metrics:
-      - label: KNN AUC
+      - label: kNN Area Under the Curve
         name: knn_auc
         summary: "Compute the kNN Area Under the Curve"
         description: |
-          "Compute the kNN Area Under the Curve"
-        reference: ""
+          "Let $f(i) \u2208 F$ be the scRNA-seq measurement of cell $i$, and $g(i) \u2208 G$ be the scATAC- seq measurement of cell $i$. kNN-AUC calculates the average percentage overlap of neighborhoods of $f(i)$ in $F$ with neighborhoods of $g(i)$ in $G$. Higher is better."
```

```suggestion
          Let $f(i) \u2208 F$ be the scRNA-seq measurement of cell $i$, and $g(i) \u2208 G$ be the scATAC- seq measurement of cell $i$. kNN-AUC calculates the average percentage overlap of neighborhoods of $f(i)$ in $F$ with neighborhoods of $g(i)$ in $G$. Higher is better.
```

No need to use quotes in combination with `|`